explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -154,19 +154,25 @@ object ObsTabTiles {
             ).zoom(Asterism.fromTargetsListOn(props.focusedTarget).asLens)
           )
 
-        val vizTimeView: Pot[View[Option[Instant]]] =
-          obsViewPot.map(_.zoom(ObsEditData.visualizationTime))
-
         val potAsterismMode: Pot[(View[Option[Asterism]], Option[ScienceMode])] =
           potAsterism.map(x => (x, scienceMode))
 
+        val vizTimeView: Pot[View[Option[Instant]]] =
+          obsViewPot.map(_.zoom(ObsEditData.visualizationTime))
+
         val vizTime = vizTimeView.toOption.flatMap(_.get)
 
-        // base coordinates corrected to vizTime
+        // asterism base coordinates at viz time or default to base coordinates
         val targetCoords: Option[CoordinatesAtVizTime] =
+          potAsterism.toOption.flatMap(_.get.map(_.baseTracking.baseCoordinates))
```

Comment:
I'm not following this. It doesn't seem to be assigned to anything so does it actually do anything?

Reply:
Uh, good catch, i restored this line by mistake

Reply:
I removed the offending line